hazelcast-client/src/test/java/com/hazelcast/client/cache/impl/nearcache/invalidation/ClientCacheInvalidationMemberAddRemoveTest.java

```diff
@@ -84,23 +82,29 @@
     private static final int RECONCILIATION_INTERVAL_SECS = 30;
     private static final int NEAR_CACHE_POPULATE_THREAD_COUNT = 5;
 
+    private HazelcastInstance secondNode;
+
     @Parameters(name = "localUpdatePolicy:{0}")
     public static Collection<Object[]> parameters() {
         return asList(new Object[][]{
                 {INVALIDATE},
-                {CACHE_ON_UPDATE},
         });
     }
 
     @Parameter
     public LocalUpdatePolicy localUpdatePolicy;
 
+    @Test
+    @Ignore("https://github.com/hazelcast/hazelcast/issues/12548")
+    public void ensure_nearCachedClient_and_member_data_sync_eventually_CACHE_ON_UPDATE() {
```

Comment:
This looks like a leftover, since there is no `CACHE_ON_UPDATE` in the parameters anyway. If you want to keep a reference to https://github.com/hazelcast/hazelcast/issues/12548, you can maybe add it as `TODO` in the `parameters()` method.

Reply:
fixed.